lib/dRep/dRepImpl.py

```diff
@@ -0,0 +1,441 @@
+# -*- coding: utf-8 -*-
+#BEGIN_HEADER
+import logging
+import os
+import sys
+import shutil
+import subprocess
+import pprint
+import uuid
+import re
+import functools
+
+from installed_clients.KBaseReportClient import KBaseReport
+from installed_clients.DataFileUtilClient import DataFileUtil
+from installed_clients.WorkspaceClient import Workspace
+from installed_clients.MetagenomeUtilsClient import MetagenomeUtils
+
+from .util import report
+from .util.dprint import dprint
+from .util.kbase_obj import BinnedContigs
+from .util.config import _globals, reset
```

Comment:
Typically a leading underscore indicates that the thing is private to the module, so would not get imported anywhere

Reply:
it looks prettier than trailing